js/plugin-loader.js

```diff
@@ -326,6 +326,10 @@ Plugin.prototype = {
             else {
               throw new Error (`No file name for data service`)
             }
+            // Make the relative path clear. process.cwd() is zlux-example-server/bin/
+            if (!path.isAbsolute(fileLocation)) {
```

Comment:
Can you please provide some details as to why this is needed? What exactly did you do so that this makes a difference?

I'm asking because I've made a similar change since this PR was created